Jenkinsfile

```diff
@@ -1,46 +1,59 @@
 
 def syncS3(String bucket, String extra_args='') {
-    try {
-        sh "bin/s3-sync.sh ${bucket} ${extra_args}"
-    } catch(err) {
-        sh "bin/irc-notify.sh --stage 's3 sync " + env.BRANCH_NAME + "' --status 'failed'"
-        throw err
-    }
-    sh "bin/irc-notify.sh --stage 's3 sync " + env.BRANCH_NAME + "' --status 'shipped'"
+  try {
+    sh "bin/s3-sync.sh ${bucket} ${extra_args}"
+  } catch(err) {
+    sh "bin/irc-notify.sh --stage 's3 sync " + env.BRANCH_NAME + "' --status 'failed'"
+      throw err
+  }
+  sh "bin/irc-notify.sh --stage 's3 sync " + env.BRANCH_NAME + "' --status 'shipped'"
 }
 
 node {
-    def image = docker.image('node:current-alpine')
+  def image = docker.image('node:current-alpine')
 
-    stage('Prepare') {
-        checkout scm
+  stage('Prepare') {
+    checkout scm
 
-        println "Pulling image ${image.id}"
-        image.pull()
+    println "Pulling image ${image.id}"
+    image.pull()
+    image.inside("-w /mdn -v ${WORKSPACE}:/mdn"){
+      sh 'npm install'
     }
+  }
 
-    stage ('Build') {
-        image.inside("-w /mdn -v ${WORKSPACE}:/mdn") {
-            sh 'rm -rf node_modules'
-            sh 'npm install && npm run build'
-        }
+  if (env.BRANCH_NAME == "master") {
+    stage('Lint') {
+      image.inside("-w /mdn -v ${WORKSPACE}:/mdn") {
+        sh 'npm run lint'
```

Comment:
This is OK for now, but I think we should add `sh 'npm run build'` here as well, since that may reveal additional errors.